### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DStatWriter**
Signed-off-by: Daniel Statham <Daniel.Statham@Broadcom.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,30 +14,38 @@ COBOL Language Support standardizes the communication between language tooling a
...
-COBOL Language Support defines the protocol that is used between an editor or IDE and a language server that provides the following COBOL syntax awareness features:
+COBOL Language Support defines the protocol that is used between an editor or IDE, and a language server that provides the following COBOL syntax awareness features:
 
 - **Autocomplete**
 
-	- Provides suggestions for COBOL keywords while you type.
+	- Provides live suggestions while you type for:
+		- COBOL keywords
+		- COBOL variables
+		- COBOL paragraphs
```
Comment (sergiuilie) [inline README.md]: please add `- Code Snippet` as well

File: README.md [hunk 2]
```diff
@@ -14,30 +14,38 @@ COBOL Language Support standardizes the communication between language tooling a
...
 		<img src="Autocomplete.gif" alt="alt text" width="500px" height="300px">
 
-- **Syntax check for keywords**
+- **Syntax and Semantic Check for Code**
 
-	- Checks for mistakes and errors in COBOL code.
+	- Checks for mistakes and errors in COBOL code. Syntax check reviews and provides suggested fixes for the whole content of the code, through syntax and semantic analysis which returns diagnosintis on the entire context of the code, not just keywords
 		<img src="SyntaxCheck.gif" alt="alt text" width="500px" height="300px">
```
Comment (sergiuilie) [inline README.md]: typo at diagnosintis

File: README.md [hunk 3]
```diff
@@ -14,30 +14,38 @@ COBOL Language Support standardizes the communication between language tooling a
...
+		
 - **Syntax highlighting** (with third-party plugins)
 	
 	- Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
```
Comment (sergiuilie) [inline README.md]: we do not need the third party to have highlighted code, it is provided by us

File: README.md (hunk 2 above)
Comment (sergiuilie) [inline README.md]: the gif is not aligned with the text

File: README.md (hunk 3 above)
Comment (sergiuilie) [reply README.md]: add syntax coloring and provide thin info there, for coloring we use third party plugin

File: README.md [hunk 4]
```diff
@@ -14,30 +14,38 @@ COBOL Language Support standardizes the communication between language tooling a
...
 	
 - **Copybook Support**
 
-	- Supports semantic analysis for variables and paragraphs across copybooks.
+	- Semantic analysis for keywords, variables, and paragraphs, across copybooks supported.
+		- Ensures and maintains compatibility of copybooks called in code.
 	- Inbuilt protection against recursive + missing copybooks.
+		- Protects against code errors where copybook may contain looping code which could crash processes.
```
Comment (sergiuilie) [inline README.md]: - throw an error when copybook missing

File: README.md [hunk 5]
```diff
@@ -14,30 +14,38 @@ COBOL Language Support standardizes the communication between language tooling a
...
 	- Variables and paragraphs defined across copybooks.
+		- Ensures consistency of code, assisting with error diagnostics as issues will not be caused by incorrect variables or paragraphs in code. 
 	- Autocomplete updated to include copybook variables and paragraphs.
+		- Speeds up coding process by intuitively suggesting most likely variables or paragraphs to follow existing code.
 	- Functionality to skip copybook levels when called, reducing call time.
 	- Inbuilt search function to identify all references filled with paragraphs and variables from copybooks
```
Comment (sergiuilie) [inline README.md]: in the entire file, I do not see those capabilities: 
-`find all references`
- `go to` definition

those two are very important features, please add it

File: README.md [hunk 6]
```diff
@@ -25,28 +25,39 @@ COBOL Language Support defines the protocol that is used between an editor or ID
...
 		<img src="SyntaxCheck.gif" alt="alt text" width="500px" height="300px">
 		
-- **Syntax highlighting** (with third-party plugins)
+- **Syntax highlighting**
 	
 	- Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
 	
+- **Syntax Coloring** (with third-party plugins)
+
+	- Contrasting colors used in displayed code for ease of identifying and distinguishing keywords, variables, code snippets and paragraphs.
```
Comment (sergiuilie) [inline README.md]: no code snippets here 👍

File: README.md [hunk 7]
```diff
@@ -25,28 +25,39 @@ COBOL Language Support defines the protocol that is used between an editor or ID
...
 
-	- Checks for mistakes and errors in COBOL code. Syntax check reviews and provides suggested fixes for the whole content of the code, through syntax and semantic analysis which returns diagnosintis on the entire context of the code, not just keywords
+	- Checks for mistakes and errors in COBOL code. Syntax check reviews and provides suggested fixes for the whole content of the code, through syntax and semantic analysis which returns diagnostics on the entire context of the code, not just keywords
+		
 		<img src="SyntaxCheck.gif" alt="alt text" width="500px" height="300px">
 		
-- **Syntax highlighting** (with third-party plugins)
+- **Syntax highlighting**
 	
 	- Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
```
Comment (sergiuilie) [inline README.md]: please remove the third party from the details as well 👍

File: README.md [hunk 8]
```diff
@@ -14,31 +14,50 @@ COBOL Language Support standardizes the communication between language tooling a
...
+		
 		<img src="SyntaxCheck.gif" alt="alt text" width="500px" height="300px">
-- **Syntax highlighting** (with third-party plugins)
+		
+- **Syntax highlighting**
 	
-	- Enables syntax highlighting for COBOL code as long as you have an appropriate third-party syntax highlighting extension installed.
+	- Enables syntax highlighting for COBOL code.
+	
+- **Syntax Coloring** (with third-party plugins)
```
Comment (zacanbrcom) [inline README.md]: I think is better be uniform to uk or us language - if in this case we choose official English please correct this to colouring

**Pull request merged**